internal/host/transition.go

```diff
@@ -380,6 +380,40 @@ func (th *transitionHandler) PostUnbindHost(sw stateswitch.StateSwitch, args sta
 		extra...)
 }
 
+////////////////////////////////////////////////////////////////////////////
+// Preparing for installation host
+////////////////////////////////////////////////////////////////////////////
+
+func (th *transitionHandler) PostPreparingForInstallationHost(sw stateswitch.StateSwitch, args stateswitch.TransitionArgs) error {
+	sHost, ok := sw.(*stateHost)
+	if !ok {
+		return errors.New("PostPreparingForInstallationHost incompatible type of StateSwitch")
+	}
+	params, ok := args.(*TransitionArgsRefreshHost)
+	if !ok {
+		return errors.New("PostPreparingForInstallationHost invalid argument")
+	}
+
+	var extra []interface{}
+	if validationFailed(params, string(models.HostValidationIDContainerImagesAvailable)) {
+		extra = append(extra, "images_status", "")
+	}
+
+	if validationFailed(params, string(models.HostValidationIDSufficientInstallationDiskSpeed)) {
+		bootDevice := hostutil.GetHostInstallationPath(sHost.host)
+		if bootDevice != "" {
```

Comment:
Any reason not to just reset the disk speed of all devices? Resetting just the boot device sounds a bit racey